.github/workflows/scripts/detect-all-changes.sh

```
@@ -0,0 +1,175 @@
+fi
+
+# Summary
+echo ""
+echo "📋 Release Summary:"
+echo "   Core: $CORE_NEEDS_RELEASE (v$CORE_VERSION)"
+echo "   Framework: $FRAMEWORK_NEEDS_RELEASE (v$FRAMEWORK_VERSION)"
+echo "   Plugins: $PLUGINS_NEED_RELEASE (${#PLUGIN_CHANGES[@]} plugins)"
+echo "   Bifrost HTTP: $BIFROST_HTTP_NEEDS_RELEASE (v$TRANSPORT_VERSION)"
+
+# Set outputs
+echo "core-needs-release=$CORE_NEEDS_RELEASE" >> $GITHUB_OUTPUT
```

Shellcheck (0.10.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)